index.js

```diff
@@ -202,15 +202,15 @@ module.exports = {
         ],
       },
     ],
+    "linebreak-style": ["error", "unix"],
+    "jsx-quotes": ["error", "prefer-single"],
+    "no-confusing-arrow": ["error", { allowParens: false }],
+    "no-tabs": ["error", { allowIndentationTabs: true }],
+    indent: ["error", 4],
+    quotes: [
+      "error",
+      "single",
+      { avoidEscape: true, allowTemplateLiterals: false },
+    ],
   },
-  "linebreak-style": ["error", "unix"],
-  "jsx-quotes": ["error", "prefer-single"],
-  "no-confusing-arrow": ["error", { allowParens: false }],
-  "no-tabs": ["error", { allowIndentationTabs: true }],
-  indent: ["error", false],
-  quotes: [
-    "error",
-    "single",
-    { avoidEscape: true, allowTemplateLiterals: false },
-  ],
```

Comment:
The prettier rules were outside of the `rules` attribute:
<img width="1461" alt="Screenshot 2024-03-01 at 12 17 58 PM" src="https://github.com/binary-com/eslint-config-deriv/assets/103016120/30c53890-48fc-428a-a020-0c3cd87d80c3">

Reply:
good catch 👍